src/App.css

```diff
@@ -1,38 +1,298 @@
+  background-color: rgb(255, 255, 255, 5%);
+  height: 3vh;
 }
 
-@keyframes App-logo-spin {
-  from {
-    transform: rotate(0deg);
+footer a {
+  font-size: 0.8rem;
+}
+
+/* BACKGROUND CHANGE CLASSES */
+
+.startingBackground {
+  background-image: url('./assets/star.jpg');
+  background-position: top center;
+  background-repeat: no-repeat;
+  height: 100vh;
+  background-size: cover;
+}
+
+.clearSky0 {
+  background-image: url('./assets/clearSky0.jpg');
+  background-position: top center;
+  background-repeat: no-repeat;
+  height: 100vh;
+  background-size: cover;
+}
+
+.clearSky1 {
+  background-image: url('./assets/clearSky1.jpg');
+  background-position: top center;
+  background-repeat: no-repeat;
+  height: 100vh;
+  background-size: cover;
+}
+
+.clearSky2 {
+  background-image: url('./assets/clearSky2.jpg');
+  background-position: top center;
+  background-repeat: no-repeat;
+  height: 100vh;
+  background-size: cover;
+}
+
+.clearSky3 {
+  background-image: url('./assets/clearSky3.jpg');
+  background-position: top center;
+  background-repeat: no-repeat;
+  height: 100vh;
+  background-size: cover;
+}
+
+.clearSky4 {
+  background-image: url('./assets/clearSky4.jpg');
+  background-position: top center;
+  background-repeat: no-repeat;
+  height: 100vh;
+  background-size: cover;
+} 
+
+.overcast {
+  background-image: url('./assets/overCastClouds.jpg');
+  background-position: top center;
+  background-repeat: no-repeat;
+  height: 100vh;
+  background-size: cover;
+}
+
+.scattered {
+  background-image: url('./assets/scatteredClouds.jpg');
+  background-position: top center;
+  background-repeat: no-repeat;
+  height: 100vh;
+  background-size: cover;
+}
+
+.broken {
+  background-image: url('./assets/brokenClouds.jpg');
+  background-position: top center;
+  background-repeat: no-repeat;
+  height: 100vh;
+  background-size: cover;
+}
+
+.few {
+  background-image: url('./assets/fewClouds.jpg');
+  background-position: top center;
+  background-repeat: no-repeat;
+  height: 100vh;
+  background-size: cover;
+}
+
+.lightRain0 {
+  background-image: url('./assets/rainy0.jpg');
+  background-position: top center;
+  background-repeat: no-repeat;
+  height: 100vh;
+  background-size: cover;
+}
+
+.lightRain1 {
+  background-image: url('./assets/rainy1.jpg');
+  background-position: top center;
+  background-repeat: no-repeat;
+  height: 100vh;
+  background-size: cover;
+}
+
+.moderateRain {
+  background-image: url('./assets/morerainy.jpg');
+  background-position: top center;
+  background-repeat: no-repeat;
+  height: 100vh;
+  background-size: cover;
+}
+
+.heavyRain {
+  background-image: url('./assets/heavyrain.jpg');
+  background-position: top center;
+  background-repeat: no-repeat;
+  height: 100vh;
+  background-size: cover;
+}
+
+.haze {
+  background-image: url('./assets/haze.jpg');
+  background-position: top center;
+  background-repeat: no-repeat;
+  height: 100vh;
+  background-size: cover;
+}
+
+.thunderstorm {
```

Comment:
Awesome classes for all the weathers! I know how the image will look like before even seeing it.